MediaBrowser.Controller/Extensions/ConfigurationExtensions.cs

```diff
@@ -120,14 +120,6 @@ public static bool GetFFmpegSkipValidation(this IConfiguration configuration)
         public static bool GetFFmpegImgExtractPerfTradeoff(this IConfiguration configuration)
             => configuration.GetValue<bool>(FfmpegImgExtractPerfTradeoffKey);
 
-        /// <summary>
-        /// Gets a value indicating whether playlists should allow duplicate entries from the <see cref="IConfiguration"/>.
-        /// </summary>
-        /// <param name="configuration">The configuration to read the setting from.</param>
-        /// <returns>True if playlists should allow duplicates, otherwise false.</returns>
-        public static bool DoPlaylistsAllowDuplicates(this IConfiguration configuration)
-            => configuration.GetValue<bool>(PlaylistsAllowDuplicatesKey);
```

Comment:
Doesn't `PlaylistsAllowDuplicatesKey` need to be removed as well?

Reply:
Yes

Reply:
Now the build is failing, I think you need to remove some more stuff?